src/gwt/src/org/rstudio/studio/client/workbench/views/source/Source.java

```diff
@@ -2633,16 +2635,6 @@ public void onSuccess(CodeBrowserEditingTarget target)
    @Override
    public void onCodeBrowserFinished(final CodeBrowserFinishedEvent event)
    {
-      tryExternalCodeBrowser(event.getFunction(), event, new Command()
-      {
-         @Override
-         public void execute()
-         {
-            final String path = CodeBrowserEditingTarget.getCodeBrowserPath(
-                  event.getFunction());
-            columnManager_.closeTabWithPath(path, false);
-         }
-      });
```

Comment:
Not sure why this code wasn't working before, but we now write in relevant "close on debugger exited" handlers elsewhere.